styles/global.js

```diff
@@ -23,13 +23,53 @@ export default css.global`
   html {
     background: white;
     box-sizing: border-box;
-    padding: 2% 5%;
+    //padding: 2% 5%;
```

Comment:
What is this doing? I've never used `styled-jsx` before.

Reply:
I commented out the padding for the HTML since I wasn't sure whether there was a reason for it being there. It was basically adding white space around the whole page!

Reply:
What do you think uncommenting it or deleting it? Commented out code is confusing cause then every person that sees it has to figure out why that code is commented out.

Reply:
Got it, I'll delete it in the next commit!